src/biz/collab/ops.rs

```diff
@@ -479,6 +483,25 @@ pub async fn list_database_row(
   Ok(db_rows)
 }
 
+pub async fn list_database_row_ids_updated(
+  collab_storage: &CollabAccessControlStorage,
+  pg_pool: &PgPool,
+  workspace_uuid_str: &str,
+  database_uuid_str: &str,
+  after: &DateTime<Utc>,
+) -> Result<Vec<DatabaseRowUpdatedItem>, AppError> {
+  let row_ids = list_database_row_ids(collab_storage, workspace_uuid_str, database_uuid_str)
```

Comment:
maybe you can collect as &str

Reply:
I can collect and make `row_ids` into `Vec<&str>`, but the following function `select_last_updated_database_row_ids` requires `&[String]` for `row_ids`, which then will require a clone. Based on what `into_iter` means, it will own the returned values from `list_database_row_ids`, which doesn't seem to require cloning for the strings.